src/features/defi/providers/cosmos/components/CosmosManager/Deposit/components/Status.tsx

```diff
@@ -67,10 +67,10 @@ export const Status = () => {
   )
 
   const handleViewPosition = useCallback(() => {
-    browserHistory.push('/wallet/earn')
-  }, [browserHistory])
+    navigate('/earn')
```

Comment:
thinking: now that we are not using the full route paths anymore, it's not clear under which route umbrella we are, we definitely need to use enums everywhere now as often as possible

Reply:
Conceptually agree, but not sure whether or not it makes sense for the sole purpose of earn - another issue with this is we have `/wallet/earn` (explicit) and `earn` (implicit subroute) so would inevitably use an enum for one but not the other.

How would you see this enum?

Reply:
Any thought I had while thinking about your question seemed like an over complication, nothing blocking this PR anyway so happy to resolve this comment and think about it during our journey